src/xpi.js

```diff
@@ -0,0 +1,80 @@
+import yauzl from 'yauzl';
+
+
+export default class Xpi {
+
+  constructor(filename, zipLib=yauzl) {
+    this.filename = filename;
+    this.zipLib = zipLib;
+    this.metadata = null;
```

Comment:
Yeah this is all about intent. I chose to use null because it's not 'undefined' if that makes sense?